data/coalition/coalition missions.txt

```diff
@@ -3211,3 +3211,604 @@ mission "Heliarchs Buy Jump Drive Mission"
+			# kst-1 is short for [K]imek [s]mall [t]alk 1
+			choice
+				`	"I'd prefer to be left alone right now."`
+					goto "kst-1 ignore"
+				`	"Yes, I am the human <first> <last>. Glad to meet you."`
+				`	"Well, I'm not the only alien."`
+					goto "kst-1 not only alien"
+				`	"We call ourselves humans, not aliens."`
+					goto "kst-1 endonym"
+			label "kst-1 glad to meet you"
+			`	"Same. 'Akacharra', my name is. To our fine neighborhood, for what purpose have you come here?"`
+			`	"My friend Anna Fangvin is publishing a book here," you say.`
+				goto "kst-1 anna"
+			label "kst-1 not only alien"
+			`	"If more of you, here there are, then who are they?"`
+			`	"It's just two; my friend Anna Fangvin and I. She's publishing a book in your space," you reply.`
+				goto "kst-1 anna"
+			label "kst-1 endonym"
+			`	"'Humans', you are? Know that, I never did. For correcting me and educating me on your fellow kind, I am thankful," says the Kimek. "So, the other human, who is she?"`
+			`	"Oh, that's my friend Anna Fangvin. She's here to publish her book in Coalition space," you answer.`
+			label "kst-1 anna"
+			`	"Anna Fangvin, her name is? An interesting name, it is, and her book, just as interesting it will be when it is released. But know the title of the book, I do not, so if tell me you will, appreciate it I will."`
+			choice
+				`	"It is called 'Beyond the Hyperdrive.' It could very well be years before release, but I'm glad you're taking interest."`
+				`	"It is called 'Beyond the Hyperdrive.' I think it's likely to be one of the best books of all time."`
+			`	"So, 'Beyond the Hyperdrive,' it is, you say? Thankful, I am. On its release, buy the book, I will remember to."`
+			`	The Kimek walks away after the two of you exchange farewells, leaving you to your own devices.`
+				goto "anna approaches"
+			label "kst-1 ignore"
+			`	"Apologies for invading your personal space, I give." The Kimek quickly walks away.`
+			label "anna approaches"
+			`	After a few hours, you see Anna enter the lounge and walk toward you. She takes a seat next to you.`
+			choice
+				`	"How did the interview go?"`
+			`	"They told me that due to the review process to make sure it's suitable, I'll have to wait for roughly half a year. Thankfully, they have arranged a small apartment for me to live in. You can go on your own now, but please make sure you come back. If you want to return to human space, please remember to come back and meet me."`
+			choice
+				`	"Why don't you just go back to human space?"`
+					goto stay
+				`	"I'll make sure to do that."`
+			label thanks
+			`	"Thanks again, <first> <last>. I'll forever be grateful to you for helping me. I never thought I'd be successful as an author, but when I met you, you helped me turn my career into something I thought I'd never see."`
+			choice
+				`	"Just helping out."`
+				`	"I'm happy to have helped you."`
+			`	"Again, thank you. You will not be needed anymore for the process."`
```

Comment:
I know that this probably isn't intentional, but this sounds kind of rude for her to say?